src/mono/wasm/testassets/WasmBasicTestApp/LibraryInitializerTest.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Globalization;
+using System.Threading.Tasks;
+using System.Resources;
+using System.Runtime.InteropServices.JavaScript;
+
+public partial class LibraryInitializerTest
+{
+    [JSExport]
+    public static void Run()
+    {
+        TestOutput.WriteLine("Run from LibraryInitializer");
```

Comment:
Pass the text as value from JS, it will be less confusing.

Reply:
Why do you find it confusing?

Reply:
because the method itself could be called from anything

Reply:
I have removed the extra text